pkgs/tools/system/hostctl/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildGoModule, fetchFromGitHub, lib, installShellFiles }:
+
+buildGoModule rec {
+  pname = "hostctl";
+  version = "1.0.14";
+
+  src = fetchFromGitHub {
+    owner = "guumaster";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "02bjii97l4fy43v2rb93m9b0ad8y6mjvbvp4sz6a5n0w9dm1z1q9";
+  };
+
+  vendorSha256 = "1lqk3cda0frqp2vwkqa4b3xkdw814wgkbr7g9r2mwxn85fpdcq5c";
+
+  doCheck = false;
+  buildFlagsArray = [ "-ldflags=-s -w -X github.com/guumaster/hostctl/cmd/hostctl/actions.version=${version}" ];
+
+  nativeBuildInputs = [ installShellFiles ];
+  postInstall = ''
+    $out/bin/hostctl completion bash > hostctl.bash
+    $out/bin/hostctl completion zsh > hostctl.zsh
+    installShellCompletion hostctl.{bash,zsh}
+    # replace above by following once merged https://github.com/NixOS/nixpkgs/pull/83630
```

Comment:
#83630 is merged.

Reply:
When I checked earlier today it still wasn't on master, just staging

Reply:
Thanks!

Reply:
Right, sorry I didn't check the target branch.